docs/en/guide/cache_management.md

```diff
@@ -253,4 +253,4 @@ For example, `--cache-dir` is `/data1:/data2`, where `/data1` has a free space o
 
 JuiceFS currently estimates the size of cached objects by adding up the size of all cached objects and adding a fixed overhead (4KiB), which is not exactly the same as the value obtained by the `du` command.
 
-To prevent the cache disk from being written to full, the client will try to reduce the cache usage when the file system where the cache directory is located is running out of space.
+To prevent the cache disk from being written to full, the client will try to reduce the cache usage when the file system the cache directory is located on is running out of space.
```

Comment:
```suggestion
To prevent the cache disk from being overwritten, the client will try to reduce the cache usage when the file system on which the cache directory is located is running out of space.
```

Reply:
`overwrite` 这个词比较特殊，计算机领域里它的意思是「重写」或者「覆盖」，我们要不要想想还有没有更合适的？